src/bp/ml/svm-js/core/svm.js

```diff
@@ -39,7 +39,7 @@ SVM.prototype.train = function(dataset) {
   var self = this
   this._training = true
   var dims = numeric.dim(dataset)
-  assert(dims[0] > 0 && dims[1] === 2 && dims[2] > 0, 'dataset must be an list of [X,y] tuples')
+  assert(dims[0] > 0 && dims[1] === 2 && dims[2] > 0, 'dataset must be a list of [X,Y] tuples')
```

Comment:
I think y is min case because its a scalar. X is an array of features so its noted in capital.

Reply:
Oh, I thought it was a typo, can you confirm @slvnperron

Reply:
yup frank's right here